CONTRIBUTING.md

```diff
@@ -0,0 +1,21 @@
+# So you want to contribute to pod-embedded-beta...
+Awesome! We're glad to have you!
+
+This set of guidelines will detail what you should know before contributing, as well as how to actually commit changes to the repository.
+
+## What do I need to know before I contribute?
+### Know where to get help!
+Always feel free to reach out on Slack, the #controls channel is a great place to start.
+### Get familiar with the repository!
+Take a look at the [readme](README.md) if you haven't already. It details the structure of the repository as well as how to set up your build environment.
```

Comment:
We should choose a convention for readme and stick with it. I.e readme/README/ReadMe

Reply:
let's stick with all caps?